app/Http/Requests/FilterTaskScheduleRequest.php

```
@@ -0,0 +1,36 @@
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'title' => ['nullable', 'string'],
+            'item_id' => ['nullable', 'string'],
+            'date_from' => ['nullable', 'date'],
+            'date_to' => ['nullable', 'date', 'after:date_from'],
+            'status' => ['nullable', 'string', Rule::in(TaskStatus::cases())],
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Validation\Rule' in method 'rules'. (Clean Code Rules)

(StaticAccess)